packages/xstate-test/src/testUtils.ts

```diff
@@ -0,0 +1,22 @@
+import { StatePlan } from '@xstate/graph';
```

Comment:
Refactor item: move this to the `/test/` directory instead (I will probably be confused again by this in the future 😂)